nlgeval/__init__.py

```diff
@@ -168,7 +168,7 @@ def load_glove(self):
         self.np = np
         self.glove_emb = Embedding()
 
-    def evaluate(self, ref, hyp):
```

Comment:
Can we keep the method name the same to reduce compatibility issues?

Reply:
@temporaer what do you think?

Reply:
@juharris i think f67414b is more consistent. just need to update the readme as well.